situp-api/src/main/java/com/amazon/situp/model/metrics/MetricNames.java

```diff
@@ -0,0 +1,8 @@
+package com.amazon.situp.model.metrics;
+
+public class MetricNames {
+    public static final String RECORDS_IN = "recordsIn";
+    public static final String RECORDS_OUT = "recordsOut";
+    public static final String TIME_ELAPSED = "timeElapsed";
+    public static final String DELIMITER = ".";
+}
```

Comment:
nit: Add some documentation for each metrics. I believe these metrics are common for all plugins but some are applicable only to source and some to sink. So lets update them.